contrib_bots/lib/define_bot.py

```diff
@@ -0,0 +1,81 @@
+# See readme.md for instructions on running this code.
+import logging
+import json
+import requests
+
+class DefineHandler(object):
+    '''
+    This plugin define a word that the user inputs. It
+    looks for messages starting with '@define'.
+    '''
+
+    DEFINITION_API_URL = 'https://owlbot.info/api/v1/dictionary/{}?format=json'
+    REQUEST_ERROR_MESSAGE = 'Definition not available.'
+    EMPTY_WORD_REQUEST_ERROR_MESSAGE = 'Please enter a word to define.'
+
+    def usage(DefineHandler):
+        return '''
+            This plugin will allow users to define a word. Users should preface
+            messages with "@define".
+            '''
+
+    def triage_message(DefineHandler, message):
+        # return True if we want to (possibly) response to this message
+        original_content = message['content']
+        # This next line of code is defensive, as we
+        # never want to get into an infinite loop of posting follow
+        # ups for own follow ups!
+        is_define = original_content.startswith('@define')
+
+        return is_define
+
+    def _handle_definition(DefineHandler, original_content):
+        # Remove '@define' from the message and extract the rest of the message, the
+        # word to define.
+        split_content = original_content.split(' ')
+
+        # If there are more than one word (a phrase)
+        if len(split_content) > 2:
+            return DefineHandler.EMPTY_WORD_REQUEST_ERROR_MESSAGE
```

Comment:
I think the user message should make it clear that multi words are not supported.